egs/librispeech/ASR/conformer_ctc_embedding_scale/test_subsampling.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python3
```

Comment:
I’m now sure that we want to copy the test code across experiments?

Reply:
The intention is to make the model dir itself as self-contained as possible so that it can be modified
independently, with some drawbacks that there are some duplicates.

We can use symlinks here if someone also agrees. @danpovey  what do you think.